NoExecutablesPathChecker.cs

```diff
@@ -6,18 +6,22 @@ namespace PathCleaner
 {
     internal class NoExecutablesPathChecker : IPathChecker
     {
-        private static readonly HashSet<string> executableExtensions = new HashSet<string>()
+        private static readonly HashSet<string> executableExtensions = new HashSet<string>(System.StringComparer.OrdinalIgnoreCase)
         {
             ".exe", ".com", ".bat", ".cmd", ".ps1", ".dll"
         };
 
+        public NoExecutablesPathChecker()
+        {
+            // TODO: Make the class compatible with executable file extensions defined in the PATHEXT to prevent EmptyPathChecker from producing false positives.
```

Comment:
Please remove this TODO and open it as a GitHub issue. TODO's in the code are hard to track and they imply a solution which isn't the right way to approach it.

Reply:
You're right. I'll do.